co.posit.RStudio.yaml

```diff
@@ -0,0 +1,149 @@
+id: co.posit.RStudio
+runtime: org.freedesktop.Sdk
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.openjdk21
+build-options:
+  append-path: /usr/lib/sdk/node20/bin
+  cflags: -O3 -DNDEBUG
+  cxxflags: -O3 -DNDEBUG
```

Comment:
```suggestion
```

Reply:
May I ask for the reason? And why not the FCFLAGS below, too?

Reply:
I have no idea what FCFLAGS does. but we set cflags and cxxflags properly you don't need to.

Reply:
Good to know! FCFLAGS is the analogue of CFLAGS, etc. to the Fortran compiler.